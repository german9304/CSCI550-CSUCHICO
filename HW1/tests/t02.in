10
0 6
1 2
2 3
2 5
4 7
4 9
5 7
6 8
8 9
