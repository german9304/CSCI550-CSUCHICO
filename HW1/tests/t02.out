4 9 8 6 0 
4 7 5 2 1 
4 7 5 2 
4 7 5 2 3 
4 7 5 
4 9 8 6 
4 7 
4 9 8 
4 9 
